android/app-newm/build.gradle.kts

```diff
@@ -1,3 +1,11 @@
+import com.android.build.gradle.internal.cxx.configure.gradleLocalProperties
+import org.jetbrains.kotlin.konan.properties.Properties
+
+apply(from = "../../gradle_include/compose.gradle")
+apply(from = "../../gradle_include/circuit.gradle")
```

Comment:
@newmskywalker going forward, you will only need to add this line to your Gradle file to bring in circuit dependencies